### ISSUE TITLE
Accessibility] Add aria labels to each of the merging editor split panes

### ISSUE THREAD

**Issue by jooyoungseo** [microsoft/vscode#227843]
Type: <b>Feature Request</b>

CC @meganrogge 

It seems like the merging editor is broken down into several panes, such as "left," "right," and "result."

When screen reader users are moving around between these panes, there is no label indicator where their current focus is landed. Please add an accessible label explicitly so that SR users can understand where they are currently focused.

VS Code version: Code - Insiders 1.93.0-insider (4849ca9bdf9666755eb463db297b69e5385090e3, 2024-09-04T13:13:15.344Z)
OS version: Windows_NT x64 10.0.22631
Modes:
